cmd/project.go

```diff
@@ -15,7 +16,20 @@ var (
 var projectCmd = &cobra.Command{
 	Use:   "project [options]",
 	Short: "Perform project operations",
-	Run:   projectGetCmd.Run,
+	Example: `  # Get current project,
+  odo project 
+  # Set myproject to current project,
```

Comment:
can you please add an empty line between each example?
It would be nice to keep it consistent with other places like `odo create -h`